src/components/inputs/LSelectNamed.vue

```diff
@@ -17,7 +17,7 @@ export default {
   name: 'LSelectNamed',
   props: {
     value: {
-      type: [String, Number],
+      type: [String, Number, Object],
```

Comment:
pra que isso?

Reply:
Para caso, for passar um passar um objeto como selected, ex: items sejam: [ {code: 'en', text: Inglês}, {...} ]e por algum motivo do lado de fora, precise do objeto inteiro no retorno. Por exemplo vou precisar do code na plataforma, e como o selected era via o valor no que era exibido ele não trazia